prep-javascript-functions/main.js

```diff
@@ -0,0 +1,55 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+var sum = addTwoNumbers(4, 7);
+console.log('sum', sum);
+
+function convertHoursToMinutes(hours) {
+  return hours * 60;
+}
+var minutes = convertHoursToMinutes(5);
+console.log('minutes', minutes);
+
+function getGreeting(name) {
+  return 'Hello ' + name + '!';
+}
+var Greeting = getGreeting('Nelson');
+console.log(Greeting);
+
+function addAndMultiplyBy5(num1, num2) {
+  const result = num1 + num2;
+  return result * 5;
+}
+var addAndMultiplySum = addAndMultiplyBy5(3, 5);
+console.log(addAndMultiplySum);
+
+function multiplyAndDivideBy5(num1, num2) {
+  const result = num1 * num2;
+  return result / 5;
+}
+var multiplyAndDivideResult = multiplyAndDivideBy5(10, 5);
+console.log(multiplyAndDivideResult);
+
+function subtractTwoNumbers(num1, num2) {
+  return num1 - num2;
+}
+var resultSubtractNumbers = subtractTwoNumbers(20, 5);
+console.log(resultSubtractNumbers);
+
+function getCircleCircumference(radius) {
+  return 2 * Math.PI * radius;
+}
+var circumference = getCircleCircumference(5);
+console.log(circumference);
+
+function getFullName(firstName, lastName) {
+  return firstName + lastName;
+}
+var name = getFullName('Nelson ', 'Daza');
```

Comment:
Adding a space at the end of the first name isn't ideal. People using your function wouldn't know to do that or remember to do that. The space should be added by the function as mentioned above